tests/default/_core/refresh.yaml

```diff
@@ -0,0 +1,10 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test _refresh.
+
+chapters:
+  - synopsis: Refresh all indices in the cluster.
+    path: /_refresh
```

Comment:
We should remove it from [here](https://github.com/opensearch-project/opensearch-api-specification/blob/main/tests/default/indices/refresh.yaml#L17).